docs/node-developers/contributing.mdx

```diff
@@ -14,9 +14,9 @@ A new version of Mina Docs is coming soon! This page will be rewritten.
 
 Mina is an open-source project with a mission to build an inclusive and sustainable community-driven protocol. As such, Mina welcomes contributions from anyone with a desire to further this goal.
 
-Currently, the protocol is in development and would benefit from contributions in the form of writing code, user testing, documentation, and community support. Below, you will find specific instructions for contributing in each of these domains.
+Currently, the protocol is in development and benefits from contributions in the form of writing code, user testing, documentation, and community support. For specific instructions for contributing in each of these domains, see the codebase repositories or ask in [Mina Protocol Discord](https://discord.gg/minaprotocol).
```

Comment:
I got an invalid invite on this link ;(

![image](https://github.com/o1-labs/docs2/assets/9512405/74eec6e3-6205-4578-99ce-b251e7f8579b)

Reply:
oh interesting, I researched and discovered:

> Christine Yip
>   [1 hour ago](https://o1-labs.slack.com/archives/C03DHRR4CMP/p1685549780178289?thread_ts=1685548649.367829&cid=C03DHRR4CMP)
> Thanks for flagging, Barrie. I think it's related to our server having a level 2 status atm, and the custom invite link is a level 3 perk. We need one more boost to get it back to lvl 3. Let me check if anyone at the MF has a discord nitro subscription and hasn't boosted the server yet.
>

Reply:
Thanks for looking into it! I'm approving the changes in this PR then since this is a MF issue to be solved :D

Reply:
I'll go ahead and merge since we have so many instances of the evergreen URL invite link all over